source/_integrations/remote.markdown

```diff
@@ -11,14 +11,18 @@ ha_codeowners:
 ha_integration_type: entity
 ---
 
-Keeps track of the remotes in your environment. Captures their state and allows you to control them.
+The **Remote** {% term integration%} keeps track of the remotes in your environment. Captures their state and allows you to control them.
```

Comment:
Not sure about this because typical a remote controls other stuff so maybe rephrase this a bit ?

Reply:
well, I only changed the formatting here, not the content.